api/tests/data/mocked_data/data/processor/config/areas/5.json

```diff
@@ -0,0 +1 @@
+{"occupancy_rules": [{"days": [1, 1, 0, 0, 0, 1, 1], "start_hour": "12:00", "finish_hour": "22:00", "max_occupancy": 12}, {"days": [1, 1, 0, 0, 0, 1, 1], "start_hour": "22:00", "finish_hour": "23:00", "max_occupancy": 11}]}
```

Comment:
Do we want this file checked in?

Reply:
No, I create a fixture for each test in test_area_occupancy_rules.py, to remove the file that those tests created when they were run in my local environment.